src/database/knexfile.js

```diff
@@ -6,20 +6,23 @@ import dotenv from 'dotenv'
 // directory.
 dotenv.config({ path: '../../.env' })
 
-export default {
-  client: 'pg',
-  version: '10',
-
-  connection: {
+const connection = process.env.NODE_ENV === 'preprod'
+  ? `${process.env.DATABASE_URL}?ssl=true`
```

Comment:
a bit weird, ideally our environment variables should be env agnostic, but OK to me.